mysql-test/suite/galera/r/galera_bf_bf_2.result

```diff
@@ -52,7 +53,64 @@ SET SESSION wsrep_on = 1;
 SET GLOBAL wsrep_provider_options = 'dbug=';
 SET GLOBAL wsrep_provider_options = 'signal=sync.apply_trx.start_of_apply_trx';
 SET DEBUG_SYNC = "now SIGNAL signal.ordered_commit_continue";
-Timeout in wait_condition.inc for SELECT COUNT(*) = 2 FROM child
 SET GLOBAL wsrep_provider_options = 'dbug=';
 SET GLOBAL wsrep_provider_options = 'signal=after_cert_and_catch';
 DROP TABLE child, parent_new;
+############################################################
+# Testcase:
+# t1 update trigger replication after noop update
+############################################################
+SET SESSION wsrep_sync_wait = 0;
+SET SESSION wsrep_sync_wait = 0;
+SET SESSION wsrep_sync_wait = 0;
+CREATE TABLE t1 (i INT NOT NULL AUTO_INCREMENT, d INT, PRIMARY KEY(i)); CREATE TABLE t2 LIKE t1; CREATE TRIGGER t1_on_update AFTER UPDATE ON t1 FOR EACH ROW INSERT INTO t2 (d) VALUES (NEW.d); INSERT INTO t1 VALUES (1, 1);;
```

Comment:
can you also add UPDATE t2 scenario?
Also may be have trigger on update t2 to insert into t3? :) 

I guess there can be no limits to madness :joy:

Reply:
My proposal is to keep it simple. update t2 with t2 having trigger inserting into t3  is the same case. 
Enough for now :)